### PR TITLE
feat: add `final-forms` exports

### PR DESCRIPTION
**Pull request by rodrigofd**
[BILL-X]

### Description

Add `AnyObject`, `FieldValidator`, `SubmissionErrors` export from `final-form`

### Review

- [ ] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/api-principles.md)
- [ ] Annotate all `props` in component with documentation
- [ ] Create `examples` for component
- [ ] Ensure that deployed demo has expected results and good examples
- [ ] Ensure that tests pass by running `yarn test`
- [ ] Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- [ ] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

### CODE REVIEW

File: packages/picasso-forms/src/index.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 // Final Form exports
-export { FORM_ERROR, FormApi, MutableState } from 'final-form'
+export { FORM_ERROR, FormApi, MutableState, AnyObject } from 'final-form'
```
Comment (PeterMK85) [inline packages/picasso-forms/src/index.ts:3]: we have more one sec

Comment (PeterMK85) [reply packages/picasso-forms/src/index.ts:3]: `FieldValidator`

Comment (PeterMK85) [reply packages/picasso-forms/src/index.ts:3]: `SubmissionErrors`

Comment (rodrigofd) [reply packages/picasso-forms/src/index.ts]: Done

**Review by PeterMK85 — APPROVED** [review]
I think all have been added which are used in BFA

**Review by piotrd — APPROVED** [review]

**Pull request merged**